src/client/actions/MessageReactionAdd.js

```diff
@@ -32,23 +38,23 @@ class MessageReactionAdd extends Action {
     // Verify reaction
     if (message.partial && !this.client.options.partials.includes(PartialTypes.REACTION)) return false;
     const existing = message.reactions.cache.get(data.emoji.id || data.emoji.name);
-    if (existing && existing.users.cache.has(user.id)) return { message, reaction: existing, user };
+    if (existing && existing.users.cache.has(member.user.id)) return { message, reaction: existing, member };
     const reaction = message.reactions.add({
       emoji: data.emoji,
       count: message.partial ? null : 0,
-      me: user.id === this.client.user.id,
+      me: me,
     });
     if (!reaction) return false;
-    reaction._add(user);
+    reaction._add(member);
     /**
      * Emitted whenever a reaction is added to a cached message.
      * @event Client#messageReactionAdd
      * @param {MessageReaction} messageReaction The reaction object
-     * @param {User} user The user that applied the guild or reaction emoji
+     * @param {Member} member The member that applied the guild or reaction emoji
```

Comment:
```suggestion
     * @param {GuildMember} member The member that applied the guild or reaction emoji
```
`Member` isnt a type

Reply:
also needs to be marked as optional
```diff
- {GuildMember} member
+ {GuildMember} [member]
```

Reply:
Its only optional if it gets moved to a third param (which it should imo). At the moment based on how its written, the typings are actually `{User|GuildMember} member`

Reply:
oops, i misread the diff and read 3 parameters lol

Reply:
I actually had it like this but vBread suggested I should change that, I'll change it back

Reply:
No, I suggested you change the description, not the type